### ISSUE TITLE
Perpetual protocol - Spot perpetual arbitrage opening position causes Unhandled error in background task

### ISSUE THREAD

**Issue by arnelhbot** [hummingbot/hummingbot#5890]
**Describe the bug**
Testing `Gateway` on `Dev 1.10.0` with `Spot_perpetual_arbitrage` using `Perp_ethereum_optimism`, it was observed that there was `Unhandled error in background task: 'USD'` , it was also tested with other markets such as AVAX and had the same issue of  `Unhandled error in background task: 'AVAX'` during testing. We tested on `paper_trade` as well as `live accounts` with a range of order amount values.

```
Traceback (most recent call last):
  File "/home/ubuntu/Hummingbot/pull_request/5885a/hummingbot/core/utils/async_utils.py", line 9, in safe_wrapper
    return await c
  File "/home/ubuntu/Hummingbot/pull_request/5885a/hummingbot/connector/gateway/amm/gateway_evm_perpetual.py", line 292, in _create_order
    amount = self.quantize_order_amount(trading_pair, amount)
  File "hummingbot/connector/connector_base.pyx", line 412, in hummingbot.connector.connector_base.ConnectorBase.quantize_order_amount
    return self.c_quantize_order_amount(trading_pair, amount)
  File "hummingbot/connector/connector_base.pyx", line 405, in hummingbot.connector.connector_base.ConnectorBase.c_quantize_order_amount
    order_size_quantum = self.c_get_order_size_quantum(trading_pair, amount)
  File "hummingbot/connector/connector_base.pyx", line 384, in hummingbot.connector.connector_base.ConnectorBase.c_get_order_size_quantum
    return self.get_order_size_quantum(trading_pair, order_size)
  File "/home/ubuntu/Hummingbot/pull_request/5885a/hummingbot/connector/gateway/amm/gateway_evm_amm.py", line 820, in get_order_size_quantum
    return max(self._amount_quantum_dict[base], self._amount_quantum_dict[quote])
KeyError: 'USD
```
**Steps To Reproduce**
Here are the steps to reproduce the issue (see attachments in the section below):

1. Create `Spot_Perpetual_Arb` strategy
2. Connect paper_trade or live spot API
3. Start strategy and observe log pane for error
4. You may change the `min_open_arbitrage percentage` to -1 to trigger arbitrage quicker

**Screenshots**
![image](https://user-images.githubusercontent.com/99257257/202152832-b6876d1f-e802-4c1a-9616-d916375eac27.png)
![image](https://user-images.githubusercontent.com/99257257/202196590-0d21b9c9-21b3-4e4d-bc67-9940ccfc9cf7.png)

**Attachments**
[perp_confi_logs.zip](https://github.com/hummingbot/hummingbot/files/10022759/perp_confi_logs.zip)

**Comment by rapcmia** [comment]
Severity: P2
Bounty: 50,000 (https://hummingbot.org/maintenance/bugs/)
Rationale: Bug related to a component on Perpetual Protocol not able to place order nor open position

**Comment by kadirc** [comment]
For perpetual, the traded pairs are not necessarily the tokens on the chain. For example, the pairs to trade on perpetual are requested from gateway /amm/perp/pairs , which have the USD or AVAX:

![image](https://user-images.githubusercontent.com/105316681/205516191-87394625-c92d-4bcd-be21-7629d6899407.png)

However, in the call to get get_order_size_quantum while placing the order;


However, the call to tokens = await GatewayHttpClient.get_instance().**get_tokens**(self.chain, self.network) 

self._amount_quantum_dict indeed does not have a key for 'AVAX' or 'USD' which gets initialized through the call to load_token_data() in base class init function: https://github.com/hummingbot/hummingbot/blob/b8d6b8097bfaa361a81f73d3a055d73c7f2c545f/hummingbot/connector/gateway/amm/gateway_evm_amm.py#L119:
![image](https://user-images.githubusercontent.com/105316681/205515786-1be5beee-9055-4e47-8042-3a40c07d1123.png)

tokens = await GatewayHttpClient.get_instance().**get_tokens**(self.chain, self.network) makes the request to ethereum optimism 'chain' to initialize tokens.

**Comment by JeremyKono** [comment]
Assigning this issue to @kadirc

**Comment by rapcmia** [comment]
https://github.com/hummingbot/hummingbot/pull/5923

**Comment by rapcmia** [comment]
Fix has been deployed to version 1.11.0
